packages/helix-shared-config/src/schemas/sitemap-language.schema.json

```diff
@@ -31,6 +31,14 @@
       "description": "The HREF language to use when listing alternates according to [RFC 5988](https://datatracker.ietf.org/doc/html/rfc5988).",
       "pattern": "^((?:(en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang))|((?:([A-Za-z]{2,3}(-(?:[A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-(?:[A-Za-z]{4}))?(-(?:[A-Za-z]{2}|[0-9]{3}))?(-(?:[A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-(?:[0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(?:x(-[A-Za-z0-9]{1,8})+))?)|(?:x(-[A-Za-z0-9]{1,8})+))$"
     },
+    "hreflangs": {
+      "type": "array",
+      "title": "Array of HREF Languages",
+      "description": "Array of HREF Languages",
+      "items": {
+        "$ref": "#/properties/hreflang"
+      }
+    },
```

Comment:
`hreflangs` is an array of type `hreflang`. I'm grateful if someone with deeper understanding of JSON Schema confirms that this is the way to express it